src/OrchardCore.Themes/TheBlogTheme/Views/Content-Category.Summary.liquid

```diff
@@ -1,7 +1,7 @@
 <div class="post-preview">
     {% a display_for:Model.ContentItem %}
         <h1>
-            <i class="fas {{ Model.ContentItem.Content.Category.Icon.Text }} fa-xs" aria-hidden="true"></i>
+            <i class="{{ Model.ContentItem.Content.Category.Icon.Text }} fa-xs" aria-hidden="true"></i>
```

Comment:
Why remove `fas`? It's used in a lot of places in OC.

Also for `Content-Category.liquid˙.

Reply:
https://github.com/OrchardCMS/OrchardCore/issues/11242#issuecomment-1065286678

Reply:
https://github.com/OrchardCMS/OrchardCore/issues/11242#issuecomment-1065301279

Reply:
Ah, OK hen.